api/v1beta1/neutronapi_types.go

```diff
@@ -102,6 +102,10 @@ type NeutronAPISpecCore struct {
 	// PreserveJobs - do not delete jobs after they finished e.g. to check logs
 	PreserveJobs bool `json:"preserveJobs"`
 
+	// +kubebuilder:validation:Optional
+	// Ml2Drivers - list of ml2 drivers to enable. Using ["ovn"] if not set.
```

Comment:
wondering if can't we set default here itself or as part of webook instead of setting in GetMl2Drivers?